src/scss/theme.scss

```diff
@@ -392,11 +402,18 @@ div.prev-next-bottom {
 /* Header Styling */
 
 h1 {
-  font-size: 3rem;
+  font-size: 2.5rem;
 }
 
 h2 {
-  font-size: 2.5rem;
+  font-size: 1.75rem;
```

Comment:
As we discussed and I included on issue #7 , I thought we agreed on sliding down all of the header sizes one level, not just h1 and h2.